ACPs/23-p-chain-native-transfers.md

```diff
@@ -0,0 +1,58 @@
+```text
+ACP: <PR Number>
+Title: P-chain native transfers
+Author(s): Dhruba Basu <https://github.com/dhrubabasu>
+Discussions-To: <GitHub Discussion URL>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Support native transfers on P-chain.
+
+## Motivation
+
+Currently, the P-chain has no simple transfer transaction type. The X-chain supports this functionality through a `BaseTx`. Although the P-chain contains transaction types that extend `BaseTx`, the `BaseTx` transaction type itself is not a valid transaction.
+
+This ACP details the functionality of the new transaction type. Activation of `BaseTx` on the Avalanche Network must be done in a mandatory upgrade.
```

Comment:
I'm not sure this is required here. We can contain this to the `Backwards Compatibility` section IMO.

Reply:
Moved.